dev/integration_tests/ios_platform_view_tests/test_driver/main_test.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
+
+Future<void> main() async {
+  FlutterDriver driver;
+
+  setUpAll(() async {
+    driver = await FlutterDriver.connect();
+  });
+
+  tearDownAll(() => driver.close());
+
+  test('Merge thread to create and remove platform views should not crash', () async {
+
+    final SerializableFinder platformViewButton =
+        find.byValueKey('platform_view_button');
+    await driver.waitFor(platformViewButton);
+    await driver.tap(platformViewButton);
+
+    final SerializableFinder plusButton =
+        find.byValueKey('plus_button');
+    await driver.waitFor(plusButton);
+    await driver.waitUntilNoTransientCallbacks();
+    await driver.tap(plusButton);
```

Comment:
Curious - what are we testing by hitting this button?

Reply:
Ohh, is this in order to unmerge?

Reply:
yes. Tapping button runs a series of animations which un-merges the threads. I will update some comments here.